src/main/java/com/rocana/lucene/codec/v1/RocanaBlockTreeTermsReader.java

```diff
@@ -228,8 +250,8 @@ public BlockTreeTermsReader(PostingsReaderBase postingsReader, SegmentReadState
           throw new CorruptIndexException("invalid sumTotalTermFreq: " + sumTotalTermFreq + " sumDocFreq: " + sumDocFreq, termsIn);
         }
         final long indexStartFP = indexIn.readVLong();
-        FieldReader previous = fields.put(fieldInfo.name,       
-                                          new FieldReader(this, fieldInfo, numTerms, rootCode, sumTotalTermFreq, sumDocFreq, docCount,
+        RocanaFieldReader previous = fields.put(fieldInfo.name,       
+                                          new RocanaFieldReader(this, fieldInfo, numTerms, rootCode, sumTotalTermFreq, sumDocFreq, docCount,
```

Comment:
We're an extremely minor fork of Lucene, ultimately just trying to comment out one line of code.

Ideally we'd stay as close to the Lucene code as possible so we can diff to see **exactly** what changed in our fork. I've used this style before and it's very nice. Even if we never use it we have a scientific way of determining exactly what we changed, no matter how much or little we diverge in the future. We could also generate a patch from this diff, to apply onto a newer version of Lucene. I'm not recommending we do that, just saying it's possible.

I vote we don't adhere to Rocana's standard as that would muddy the diff.